packages/account-v2/package.json

```diff
@@ -11,7 +11,7 @@
         "start": "rimraf dist && npm run test && npm run serve"
     },
     "dependencies": {
-        "@deriv-com/ui": "1.8.1",
+        "@deriv-com/ui": "1.8.2",
```

Comment:
@shahzaib-deriv @thisyahlen-deriv is the checkbox issue resolved in the 1.18.2 ?

Reply:
Yeaps

Reply:
Yes @mahdiyeh-deriv the checkbox issue is fixed with the new version of deriv-ui